src/index.ts

```diff
@@ -38,12 +44,24 @@ export const beautifier: Beautifier = {
     TypeScript: options.Script,
     Vue: options.Script,
   },
+  resolveConfig: ({ filePath, dependencies }) => {
+    if (!filePath) {
+      return Promise.resolve({});
```

Comment:
This line should be covered.  Think you just need a test where you pass `prefer_beautifier_config: true` but without a filePath.

Reply:
Good point. On it!

Reply:
Done.